test/unit/android-helper-specs.js

```diff
@@ -383,6 +383,22 @@ describe('Android Helpers', () => {
       mocks.adb.verify();
     });
   }));
+  describe('setMockLocationApp', withMocks({adb}, (mocks) => {
+    it('should enable mock location for api level below 23', async () => {
+      mocks.adb.expects('getApiLevel').returns("18");
```

Comment:
I think this line (and the one in the other test) should read:

```js
mocks.adb.expects('getApiLevel').returns(Promise.resolve("18"));
```

because `getApiLevel` is an async function and should return a Promise.